### DISCUSSION TITLE
Confused by the bundling of JS.

### DISCUSSION THREAD

**Discussion by r146023** [jgraph/drawio-desktop#597] [Q&A]
Hello, I am confused by how the app.min.js and other bundled files are produced?
I want to modify the underlying JS but the app.min.js is never updated, and there doesn't appear to be any bundling occurring at all.
I can go into the app.min.js and change things that actually effect the program, but that is obviously not the ideal way to handle it.

Perhaps I have went about this all wrong, I am not sure.

**Comment by 0xorial** [comment]
Maybe this should be covered in readme?..
